server/src/infra/auth_service.rs

```diff
@@ -352,19 +352,24 @@ where
     let token = create_jwt(data.get_tcp_handler(), &data.jwt_key, name, groups).await;
     let refresh_token_plus_name = refresh_token + "+" + name.as_str();
 
+    let mut path = data.server_url.path().to_string();
+    if path != "/" {
```

Comment:
If the URL ends with a slash, is it trimmed when calling path()? Or could you end up with 2 slashes here?

It seems more straightforward to say `if !path.ends_with("/")`